content/articles/what-is-ssl-san.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: What is the Subject Alternative Name (SAN)?
+excerpt: The Subject Alternative Name (SAN) is an extension to the X.509 specification that allows to specify additional host names for a single SSL certificate.
+categories:
+- SSL Certificates
+---
+
+# What is the SSL Certificate Subject Alternative Name?
```

Comment:
I'd consider swapping this section with the "What is a SAN certificate" section beneath it. We link to this page from other places in the docs, typically a user is going to click on those links with the question "what's a SAN certificate" in mind. I feel like diving straight into the background and the X.509 specification might be a little much to lead off with.